README.md

```diff
@@ -38,7 +38,7 @@ Configure the Magento instance
   1. Database user should be ```magento```
   1. Database password should be ```magento```
 2. In 'Web access options'
-  1. Enter a Base URL of ```https://magento.dev```.
+  1. Enter a Base URL of ```http://magento.dev```.
```

Comment:
When I originally put this together, it was to troubleshoot issues integrating with magento using TLS for SSL as a matter of PCI compliance.  Is there a specific reason we no longer want to emulate a production setup with PCI compliant SSL?

Reply:
I didn't consider PCI compliance testing here. That said, and this might be something specific to my setup, but attempting to reach `https://magento-dev` with the original README instructions results in a "Too many redirects" error and doesn't _appear_ to be going through nginx (at least based on the output from docker-compose).

Reply:
Actually, the requests *are* going through nginx. I can see them in nginx's magento.log file, but my change may have jacked something up in relation to the nginx configuration or more likely the SSL cert. I'm getting an unstyled page when accessing the site with SSL and the browser console shows a number of errors like:

```
[Warning] [blocked] The page at https://magento.dev/ was not allowed to run insecure content from http://magento.dev/js/prototype/prototype.js.
```